python/lsst/obs/lsstSim/lsstSimMapper.py

```diff
@@ -134,7 +135,7 @@ def validate(self, dataId):
             if component not in dataId:
                 continue
             val = dataId[component]
-            if not isinstance(val, str):
+            if not isinstance(val, basestring):
```

Comment:
I was following https://sqr-014.lsst.io/, and this did pass Jenkins on both python 2 and 3. Is there a different test I should do?